src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -646,9 +639,11 @@ public async Task<ContentItem> CloneAsync(ContentItem contentItem)
             context.CloneContentItem.DisplayText = contentItem.DisplayText;
 
             await Handlers.InvokeAsync((handler, context) => handler.CloningAsync(context), context, _logger);
-            await ReversedHandlers.InvokeAsync((handler, context) => handler.ClonedAsync(context), context, _logger);
 
             _session.Save(context.CloneContentItem);
```

Comment:
Before `ClonedAsync()` handlers as it has not been queried, so not yet in the identity map